src/picongpu/include/particles/particleToGrid/ComputeGridValuePerFrame.def

```diff
@@ -143,7 +219,22 @@ struct CreateMacroCounterOperation
     typedef FieldTmpOperation< MacroParticleCounter, T_Species > type;
 };
 
-/* EnergyDensity */
+/** Energy Density Operation for Particle to Grid Projections
+ *
+ * Derives a scalar field for average kinetic particle energy per cell times the
+ * particle density from a particle species at runtime.
```

Comment:
QUESTION: 
Would `kinetic particle energy density` be not more straight forward?